src/dvc_objects/fs/base.py

```diff
@@ -646,18 +632,14 @@ def get_file(rpath, lpath, **kwargs):
             ]
 
         jobs = batch_size or self.jobs
-        if self.fs.async_impl:
-            return self.fs.get(
-                from_infos,
-                to_infos,
-                callback=callback,
-                batch_size=jobs,
-            )
-
-        callback.set_size(len(from_infos))
-        executor = ThreadPoolExecutor(max_workers=jobs, cancel_on_error=True)
-        with executor:
-            list(executor.imap_unordered(get_file, from_infos, to_infos))
+        copy(
```

Comment:
we could use `transfer` here to get reflinking instead of `copy` in the event we are only doing local->local `get()` but using `copy` keeps us consistent with existing `get()/put()` behavior